hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoopImpl/mapred/AccumuloRecordReader.java

```diff
@@ -161,7 +161,7 @@ public void initialize(InputSplit inSplit, JobConf job) throws IOException {
       } catch (TableNotFoundException e) {
         throw new IOException(e);
       }
-
```

Comment:
Could possibly call setConsistencyLevel once on the scannerBase var after the if/else .. could also do this in the mapreduce pkg

Reply:
As currently implemented, it sets the consistency level on the BatchScanner and Scanner. Your suggestion would also set it on the OfflineScanner, where the call may work, but will have no effect. This has not been tested.